--- frontends/SConscript.tcl	2009-02-17 08:04:08.000000000 +0100
+++ frontends/SConscript	2011-05-19 05:50:34.000000000 +0200
@@ -281,9 +281,11 @@
     if util.platform == "linux":
         tclIncludePath = Split('''
             /usr/include/tcl
+            /usr/include/tcl8.6
             /usr/include/tcl8.5
             /usr/include/tcl8.4
             /usr/include/tk
+            /usr/include/tk8.6
             /usr/include/tk8.5
             /usr/include/tk8.4
         ''')
@@ -305,11 +307,11 @@
     csTclEnvironment = commonEnvironment.Clone()
     csTclEnvironment.Append(CPPPATH = tclIncludePath)
 
-    # Look for tcl and tk libs, try 8.5 first and then 8.4
+    # Look for tcl and tk libs, try 8.6 first and then 8.5
     tcllibfound = False
     tklibfound = False
     if util.platform == "linux" or util.platform == "win32":
-        for version in ['8.5', '85', '8.4', '84']:
+        for version in ['8.6', '86', '8.5', '85']:
             tcllibfound = tcllibfound or configure.CheckLib("tcl%s" % version,
                     autoadd=0)
             tklibfound = tklibfound or configure.CheckLib("tk%s" % version,
--- SConstruct.tcl	2011-05-19 05:50:34.000000000 +0200
+++ SConstruct	2011-05-19 05:52:21.000000000 +0200
@@ -264,8 +264,8 @@
     'Set to 1 to build OLPC version',
     '0')
 commandOptions.Add('tclversion',
-    'Set to 8.4 or 8.5',
-    '8.5')
+    'Set to 8.6',
+    '8.6')
 commandOptions.Add('includeMP3',
      'Set to 1 if using mpadec',
      '0')
