diff -rupN csound-20260218.old/CMakeLists.txt csound-20260218/CMakeLists.txt
--- csound-20260218.old/CMakeLists.txt	2026-02-18 18:32:40.469908572 +0100
+++ csound-20260218/CMakeLists.txt	2026-02-18 18:43:09.849577733 +0100
@@ -1682,7 +1682,7 @@ endif()
 install(EXPORT CsoundExports
     FILE "CsoundConfig.cmake"
     NAMESPACE "Csound::"
-    DESTINATION "${CMAKE_INSTALL_PREFIX}/share/csound"
+    DESTINATION "${CMAKE_INSTALL_PREFIX}/share/cmake/csound"
 )
 
 # configure and install pkg-config file
