diff -rupN Csound5.18.02.old/SConstruct Csound5.18.02/SConstruct
--- Csound5.18.02.old/SConstruct	2012-10-31 13:34:26.000000000 +0100
+++ Csound5.18.02/SConstruct	2012-10-31 13:54:24.847504491 +0100
@@ -835,9 +835,7 @@ if not configure.CheckSndFile():
 # library version is CS_VERSION.CS_APIVERSION
 csoundLibraryVersion = '5.2'
 csoundLibraryName = 'csound'
-if commonEnvironment['useDouble'] != '0':
-    csoundLibraryName += '64'
-elif getPlatform() == 'win32':
+if getPlatform() == 'win32':
     csoundLibraryName += '32'
 # flags for linking with the Csound library
 libCsoundLinkFlags = commonEnvironment['LINKFLAGS'] 
@@ -893,8 +891,6 @@ if commonEnvironment['buildRelease'] !=
             tmp = '%s/lib/csound/plugins' % commonEnvironment['prefix']
         else:
             tmp = '%s/lib64/csound/plugins' % commonEnvironment['prefix']
-        if commonEnvironment['useDouble'] != '0':
-            tmp += '64'
         s = '-DCS_DEFAULT_PLUGINDIR=\\"%s\\"' % tmp
         csoundLibraryEnvironment.Append(CPPFLAGS = [s])
     elif buildOSXFramework != 0:
@@ -2841,10 +2837,7 @@ for i in sys.path:
     if i[:sys.prefix.__len__()] == sys.prefix and i[-13:] == 'site-packages':
         PYTHON_DIR = i
 
-if commonEnvironment['useDouble'] == '0':
-    PLUGIN_DIR = LIB_DIR + "/csound/plugins"
-else:
-    PLUGIN_DIR = LIB_DIR + "/csound/plugins64"
+PLUGIN_DIR = LIB_DIR + "/csound/plugins"
 
 if commonEnvironment['install'] == '1':
     installExecutables = Alias('install-executables',
diff -rupN Csound5.18.02.old/Top/csmodule.c Csound5.18.02/Top/csmodule.c
--- Csound5.18.02.old/Top/csmodule.c	2012-10-31 13:42:39.000000000 +0100
+++ Csound5.18.02/Top/csmodule.c	2012-10-31 13:56:46.211902408 +0100
@@ -131,7 +131,6 @@ static  const   char    *InfoFunc_Name =
 
 /* environment variable storing path to plugin libraries */
 static  const   char    *plugindir_envvar =   "OPCODEDIR";
-static  const   char    *plugindir64_envvar = "OPCODEDIR64";
 
 /* default directory to load plugins from if environment variable is not set */
 #if !(defined(_CSOUND_RELEASE_) && (defined(LINUX) || defined(__MACH__)))
@@ -685,17 +684,12 @@ int csoundLoadModules(CSOUND *csound)
       return CSOUND_ERROR;
 
     /* open plugin directory */
-    dname = csoundGetEnv(csound, (sizeof(MYFLT) == sizeof(float) ?
-                                  plugindir_envvar : plugindir64_envvar));
+    dname = csoundGetEnv(csound, plugindir_envvar);
     if (dname == NULL) {
 #if ENABLE_OPCODEDIR_WARNINGS
       csound->opcodedirWasOK = 0;
-#  ifdef USE_DOUBLE
-      dname = csoundGetEnv(csound, plugindir_envvar);
-      if (dname == NULL)
-#  endif
 #endif
-        dname = CS_DEFAULT_PLUGINDIR;
+      dname = CS_DEFAULT_PLUGINDIR;
     }
     dir = opendir(dname);
     if (UNLIKELY(dir == (DIR*) NULL)) {
