diff -rupN csound-20250719.old/installer/misc/vim/cshelp csound-20250719/installer/misc/vim/cshelp
--- csound-20250719.old/installer/misc/vim/cshelp	2025-07-22 18:30:19.374730300 +0200
+++ csound-20250719/installer/misc/vim/cshelp	2025-07-22 21:09:02.052976848 +0200
@@ -3,7 +3,9 @@
 BROWSER_CMD=""
 
 if [ "${DISPLAY}" != "" ] ; then
-    if ( which links | grep -q -G -e "^/usr" ) ; then
+    if ( which xdg-open | grep -E -e "^(/usr)?/bin" ); then
+        BROWSER_CMD="xdg-open" ;
+    elif ( which links | grep -q -G -e "^/usr" ) ; then
         BROWSER_CMD="links -g" ;
     elif ( which konqueror | grep -q -G -e "^/opt" ) ; then
         BROWSER_CMD="konqueror" ;
@@ -38,5 +40,5 @@ else
     OPNAME="${1}" ;
 fi
 
-exec ${BROWSER_CMD} "/usr/local/share/doc/csound/manual/${OPNAME}.html"
+exec ${BROWSER_CMD} "/usr/share/doc/csound/manual/${OPNAME}.html"
 
