diff -rupN csound-20260218.old/installer/misc/vim/cshelp csound-20260218/installer/misc/vim/cshelp
--- csound-20260218.old/installer/misc/vim/cshelp	2026-02-18 18:21:59.271831300 +0100
+++ csound-20260218/installer/misc/vim/cshelp	2026-02-18 18:41:47.786776643 +0100
@@ -3,7 +3,9 @@
 BROWSER_CMD=""
 
 if [ "${DISPLAY}" != "" ] ; then
-    if ( which links | grep -q -G -e "^/usr" ) ; then
+    if ( which xdg-open | grep -E -e "^(/usr)?/bin" ); then
+        BROWSER_CMD="xdg-open" ;
+    elif ( which links | grep -q -G -e "^/usr" ) ; then
         BROWSER_CMD="links -g" ;
     elif ( which konqueror | grep -q -G -e "^/opt" ) ; then
         BROWSER_CMD="konqueror" ;
@@ -38,5 +40,5 @@ else
     OPNAME="${1}" ;
 fi
 
-exec ${BROWSER_CMD} "/usr/local/share/doc/csound/manual/${OPNAME}.html"
+exec ${BROWSER_CMD} "/usr/share/doc/csound/manual/${OPNAME}.html"
 
