diff -rupN Csound5.18.02.old/SConstruct Csound5.18.02/SConstruct
--- Csound5.18.02.old/SConstruct	2012-10-31 14:02:25.000000000 +0100
+++ Csound5.18.02/SConstruct	2012-10-31 14:32:37.937909824 +0100
@@ -261,8 +261,8 @@ commandOptions.Add('useOpenMP',
     'Set to 1 to use OpenMP for parallel performance',
     '0')
 commandOptions.Add('tclversion',
-    'Set to 8.4 or 8.5',
-    '8.5')
+    'Set to 8.6',
+    '8.6')
 commandOptions.Add('includeWii',
      'Set to 12 or 13 if using libwiiuse v0.12 or v0.13',
      '0')
