diff -rupN Csound5.18.02.old/Top/csound.c Csound5.18.02/Top/csound.c
--- Csound5.18.02.old/Top/csound.c	2012-08-31 15:40:34.000000000 +0200
+++ Csound5.18.02/Top/csound.c	2012-10-31 14:14:13.422856019 +0100
@@ -1040,7 +1040,7 @@ extern "C" {
       max_len = 21;
       csoundCreateGlobalVariable(p, "_RTAUDIO", (size_t) max_len);
       s = csoundQueryGlobalVariable(p, "_RTAUDIO");
-      strcpy(s, "PortAudio");
+      strcpy(s, "pulse");
       csoundCreateConfigurationVariable(p, "rtaudio", s, CSOUNDCFG_STRING,
                                         0, NULL, &max_len,
                                         "Real time audio module name", NULL);
