diff -rupN csound-20250719.old/CMakeLists.txt csound-20250719/CMakeLists.txt
--- csound-20250719.old/CMakeLists.txt	2025-07-22 20:28:21.492143150 +0200
+++ csound-20250719/CMakeLists.txt	2025-07-22 23:38:13.595047226 +0200
@@ -1637,7 +1637,7 @@ endif()
 install(EXPORT CsoundExports
     FILE "CsoundConfig.cmake"
     NAMESPACE "Csound::"
-    DESTINATION "${CMAKE_INSTALL_PREFIX}/share/csound"
+    DESTINATION "${CMAKE_INSTALL_PREFIX}/share/cmake/csound"
 )
 
 # install samples
