diff -rupN Csound5.18.02.old/Top/csmodule.c Csound5.18.02/Top/csmodule.c
--- Csound5.18.02.old/Top/csmodule.c	2012-08-31 15:40:34.000000000 +0200
+++ Csound5.18.02/Top/csmodule.c	2012-10-31 13:42:39.675398298 +0100
@@ -136,17 +136,20 @@ static  const   char    *plugindir64_env
 /* default directory to load plugins from if environment variable is not set */
 #if !(defined(_CSOUND_RELEASE_) && (defined(LINUX) || defined(__MACH__)))
 #  define ENABLE_OPCODEDIR_WARNINGS 1
-#  ifdef CS_DEFAULT_PLUGINDIR
-#    undef CS_DEFAULT_PLUGINDIR
+#  ifndef CS_DEFAULT_PLUGINDIR
+#    ifndef USE_DOUBLE
+#      define CS_DEFAULT_PLUGINDIR  "/usr/lib/csound/plugins"
+#    else
+#      define CS_DEFAULT_PLUGINDIR  "/usr/lib64/csound/plugins"
+#    endif
 #  endif
-#  define CS_DEFAULT_PLUGINDIR      "."
 #else
 #  define ENABLE_OPCODEDIR_WARNINGS 0
 #  ifndef CS_DEFAULT_PLUGINDIR
 #    ifndef USE_DOUBLE
-#      define CS_DEFAULT_PLUGINDIR  "/usr/local/lib/csound/plugins"
+#      define CS_DEFAULT_PLUGINDIR  "/usr/lib/csound/plugins"
 #    else
-#      define CS_DEFAULT_PLUGINDIR  "/usr/local/lib/csound/plugins64"
+#      define CS_DEFAULT_PLUGINDIR  "/usr/lib/csound/plugins64"
 #    endif
 #  endif
 #endif
